string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,11 @@
+/* exported numVowels */
+function numVowels(string) {
+  var vowels = 0;
+  for (var i = 0; i < string.length; i++) {
+    if (string[i] === 'a' || string[i] === 'e' || string[i] === 'o' || string[i] === 'u' || string[i] === 'i' ||
```

Comment:
This works but you could also use a switch statement or `includes` (something like:
`['a', 'e', 'i', 'o', 'u'].includes(string[i].toLowerCase())`